src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -412,19 +386,76 @@ private boolean candidateStringAlreadyInUseElsewhere(
     return inUse;
   }
 
+  private void validateCvrFileSources() {
+    if (rawConfig.cvrFileSources == null || rawConfig.cvrFileSources.isEmpty()) {
+      isValid = false;
+      Logger.log(Level.SEVERE, "Contest config must contain at least 1 cast vote record file!");
+    } else {
+      HashSet<String> cvrFilePathSet = new HashSet<>();
+      for (CVRSource source : rawConfig.cvrFileSources) {
+        if (!isCvrSourceValid(source)) {
```

Comment:
I also find this method name (`isCvrSourceValid`) misleading. You actually validate a bunch of additional stuff related to the source below, outside of that method.

Reply:
Yeah, but that forces it to be aware of what's going on elsewhere in the config, and thus it can't be `static` anymore, which is what I need. Could I just rename this to something like `initialCvrSourceValidationCheck`?

Reply:
Yeah, I'm fine renaming it. I didn't mean that you needed to change its behavior.